src/index.css

```diff
@@ -141,62 +209,69 @@ a:focus {
 
 .label {
   display: block;
-  font-size: 10px;
+  font-family: Montserrat, sans-serif;
+  font-size: 1.25rem;
   font-weight: bold;
-  color: #8f8888;
-  padding: 10px 0 4px 0;
-  margin: 0;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: -0.18px;
+  color: #888;
 }
 
 /* .form-box input */
 input[type='text']:nth-child(1) {
   width: 56%;
-  font-size: 10px;
-  padding: 4px;
-  color: #333;
-  border: 1px solid #e9e9e9;
-  border-radius: 2px;
+  padding: 0.75rem;
+  border-radius: 4px;
+  border: solid 1px #e8e8e8;
+  background-color: #fff;
 }
 
 input[type='text']:nth-child(2) {
   width: 24%;
-  font-size: 10px;
-  color: #333;
-  padding: 4px;
   margin-left: 2.5%;
-  border: 1px solid #e9e9e9;
-  border-radius: 2px;
+  padding: 0.75rem;
+  border-radius: 4px;
+  border: solid 1px #e8e8e8;
+  background-color: #fff;
 }
 
 input[type='button'],
 input[type='submit'],
 input[type='reset'] {
-  font-size: 8px;
+  width: 15%;
+  margin-left: 2.5%;
+  padding: 1.1rem;
+  border-radius: 3px;
   background-color: #0290ff;
+  font-family: RobotoSlab, sans-serif;
+  font-size: 0.813rem;
+  font-weight: bold;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
```

Comment:
As these three lines are getting repeated in different classes. The best practice is to create a new class and place these properties there instead of repeating the properties over and over again. It'll save your CSS lines as well 👍🏻 😄 
#### For Example
```css
.class-name {
  font-stretch: normal;
  font-style: normal;
  line-height: normal;
}
```